views/random-beer.hbs

```diff
@@ -0,0 +1,18 @@
+
+<div>
+    <div class="beers-container">
+        {{#each radomBeer}}
+            {{> beerpartial this}}
```

Comment:
the name of the partial here `beerpartial` is different from the name of the partial file `beerPartial`